apps/src/javalab/JavalabEditor.jsx

```diff
@@ -504,9 +515,14 @@ class JavalabEditor extends React.Component {
             isRtl={false}
             label={javalabMsg.newFile()}
             leftJustified
+            isDisabled={isReadOnlyWorkspace}
           />
           <PaneSection style={styles.backpackSection}>
-            <Backpack isDarkMode={isDarkMode} />
+            <Backpack
+              id={'javalab-editor-backpack'}
```

Comment:
What does this ID get used for?

Reply:
Just in a unit test -- the other three buttons in the header have IDs, and I thought it made more sense to be able to loop through IDs than have 3 with IDs and one with some other Enzyme selector:

https://github.com/code-dot-org/code-dot-org/blob/a4014906ac25d5e897dbb309993b38ea8d2406e4/apps/test/unit/javalab/JavalabEditorTest.js#L60